File: modulo01/mario.c
#include <stdio.h>
#include "cs50.h"

int main(void)
{
    int altura;
    do
    {
        altura = get_int("Altura: ");
    } while ((altura < 1) || (altura > 8));
    
    for (int n = 1; n <= altura; n++)
    {
        for (int v = 1; v <= altura - n; v++)
        {
            printf(" ");
        }
                
        for (int c = 1; c <= n; c++)
        {
            printf("#");
        }

        printf(" ");

        for (int c = 1; c <= n; c++)
        {
            printf("#");
        }
            
        printf("\n");
    }
}

File: pset6/mario.py
import cs50

altura = cs50.get_int("Digite a altura da pirâmide: ")

for i in range(altura):
    for a in range(altura - i - 1):
        print(" ", end="")
    for b in range(i + 1):
        print("#", end="")
    for c in range(i + 1):
        print("#", end="")
    print()